src/iotjs_debuglog.h

```diff
@@ -36,6 +37,8 @@ extern const char* iotjs_debug_prefix[4];
       fprintf(iotjs_log_stream, __VA_ARGS__);                       \
       fprintf(iotjs_log_stream, "\n");                              \
       fflush(iotjs_log_stream);                                     \
+      if (iotjs_debug_extern_out)                                   \
+        iotjs_debug_extern_out(lvl, __VA_ARGS__);                   \
```

Comment:
@daeyeon What is the difference between `iotjs_log_stream` and `iotjs_debug_extern_out`? 

First, I cannot understand why do we need two separate logging systems.

Second, implementation are scattered in inconsistent way. `iotjs_log_stream` is defined as global variable in iotjs_debuglog.c and `iotjs_debug_extern_out` is defined in iotjs_env as a member. It makes me understand and follow up the code.